policy/release/cve/cve.rego

```diff
@@ -256,6 +226,45 @@ deny contains result if {
 	result := lib.result_helper_with_severity(rego.metadata.chain(), [e.message], e.severity)
 }
 
+# Create a data structure that represents the different vulnerabilities sorted in different
+# buckets, making it easier to implement the different rules.
+# _grouped_vulns := {
+# 	"restrict_cve_security_levels": {
+# 		"critical": {vuln1, vuln2}
+# 	},
+# 	"restrict_unpatched_cve_security_levels": {
+# 		"critical": {vuln3},
+# 		"high": {vuln4},
+# 	}
+# }
+_grouped_vulns[key][level] contains vuln if {
+	some key, patched in _levels_keys_patched
+	levels := lib.rule_data(key)
+	some vuln in _clair_report.vulnerabilities
```

Comment:
Is this making the same network call for each entry in `_levels_keys_patched`? If so, I'm wondering if it can be made once to reduce chance for errors and the time it takes?

Reply:
No, not every time, but it's happening twice which I don't know why.

I built a custom version of the EC binary that just emits a loud message when a manifest and blob are fetched. I tried different approaches and can't seem to make it just a single call. I also noticed that this happens to multiple other refs. We should probably do something about that.

That is unrelated to this PR though.

Reply:
Rego does some caching when it comes to rule resolution. My guess is that there are a couple of queries that run in parallel before the caching kicks in.